qiskit/circuit/quantumcircuit.py

```diff
@@ -2817,7 +2828,7 @@ def _unroll_param_dict(
         self, value_dict: Mapping[Parameter, ParameterValueType]
     ) -> dict[Parameter, ParameterValueType]:
         unrolled_value_dict: dict[Parameter, ParameterValueType] = {}
-        for (param, value) in value_dict.items():
+        for param, value in value_dict.items():
```

Comment:
Can we revert unrelated formatting changes?

Reply:
I can try. I don't recall making that change myself. Maybe ruff?

Reply:
Done in c062ca6